object/virtual_device_list.go

```diff
@@ -57,6 +57,18 @@ func SCSIControllerTypes() VirtualDeviceList {
 	})
 }
 
+// SATAControllerTypes are used for adding a new SATA controller to a VM.
+func SATAControllerTypes() VirtualDeviceList {
```

Comment:
Don't see this function used anywhere, seems we don't need it?

Reply:
It's used in the `TestCreateSATAController` in the tests.

Reply:
Resolved in 2e77836.